Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.yml

```diff
@@ -0,0 +1,28 @@
+args:
+- description: IP address.
+  name: ip_address
+  required: true
+- default: true
+  defaultValue: "3"
+  description: Request timeout value, in seconds. Default value is 3 seconds.
+  name: time_out
+comment: Checks to see if SNMP is enabled on the IP address provided.
+commonfields:
+  id: SnmpDetection
+  version: -1
+dockerimage: demisto/python3:3.10.9.46032
+enabled: true
+name: SnmpDetection
+outputs:
+- contextPath: SnmpDetection
+  description: Checks if SNMP version is enabled and gets the versions running
```

Comment:
Does this output really get the version running?  Or does it just display whether the SNMP version is enabled.